### PR TITLE
47034_View_Staff_Details

### PR DESCRIPTION
**Pull request by AVI5HEK**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -35,6 +35,8 @@
             android:label="@string/title_activity_staff_detail"
             android:theme="@style/MyMaterialTheme" >
         </activity>
+        <activity android:name=".activity.StaffListActivity" >
```
Comment (nvmanh) [inline app/src/main/AndroidManifest.xml]: can we use "<activity android:name=".activity.StaffListActivity" />"???

Comment (AVI5HEK) [inline app/src/main/AndroidManifest.xml]: I don't understand what you asked. Could you please clarify?

Comment (nvmanh) [inline app/src/main/AndroidManifest.xml]: ok, ignore it

**Comment by nvmanh** [comment]
OK!

**Pull request merged**